helm_deploy/hmpps-education-and-work-plan-ui/values.yaml

```diff
@@ -61,7 +61,9 @@ generic-service:
     UPDATE_INDUCTION_PRE_PRISON_EDUCATION_SECTION_ENABLED: "false"
     NEW_COURSE_AND_QUALIFICATION_HISTORY_ENABLED: "false"
     NEW_CREATE_GOAL_JOURNEY_ENABLED: "false"
+    AUDIT_SQS_QUEUE_URL: "sqs_queue_url"
```

Comment:
Minor, and I'm sure it works fine, but does this need to be here (line 64) as well as in the secrets (line 85)?